### PR TITLE
Fix a bunch of places we still referred to applications as 'service'

### PR DESCRIPTION
**Pull request by jameinel**
## Description of change

Simply a code cleanup.

## QA steps

Nothing should be impacted, it is just internal renaming.

## Documentation changes

No

## Bug reference

None

### CODE REVIEW

File: state/application.go [hunk 1]
```diff
@@ -1234,7 +1234,7 @@ func (a *Application) removeUnitOps(u *Unit, asserts bson.D) ([]txn.Op, error) {
 	if err != nil {
 		return nil, err
 	}
-	resOps, err := removeUnitResourcesOps(a.st, u.doc.Application, u.doc.Name)
```
Comment (anastasiamac) [inline state/application.go]: Is application not needed in this call?

**Comment by jameinel** [comment]
Nothing in the function referred to it.

John
=:->

**Comment by jameinel** [comment]
$$merge$$

**Comment by anastasiamac** [comment]
Niice :D

**Comment by jameinel** [comment]
Failures look spurious
$$merge$$

**Pull request merged**